rapid7_insightidr/plugin.spec.yaml

```diff
@@ -599,22 +599,61 @@ actions:
         type: string
         example: Firewall Activity
         required: true
+    output:
+      results:
+        title: Query Results
+        description: Query Results
+        type: "[]events"
+        required: true
+
+  advanced_query_on_log_set:
+    title: Advanced Query on Log Set
+    description: Realtime query into an Insight IDR log set
+    input:
+      query:
+        title: Query
+        description: Query
+        type: string
+        example: "where(user=adagentadmin, loose)"
+        required: true
+      time_from:
+        title: Time From
+        description: Beginning time and date for the query
+        type: string
+        example: "01-01-2020"
```

Comment:
This example doesn't have a time but states a date and time is supported. As a user I also can't tell how to specify a time and if a time is required.